<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Load"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Compile"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetFragments">
   <property name="windowTitle">
    <string>Fragments</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListView" name="listViewFragments"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSamplers">
   <property name="windowTitle">
    <string>Samplers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="listViewSamplers"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="materialeditor.qrc">
     <normaloff>:/icons/icons/Save.png</normaloff>:/icons/icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="icon">
    <iconset resource="materialeditor.qrc">
     <normaloff>:/icons/icons/Load.png</normaloff>:/icons/icons/Load.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="toolTip">
    <string>Load</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="materialeditor.qrc">
     <normaloff>:/icons/icons/Exit.png</normaloff>:/icons/icons/Exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Compile">
   <property name="icon">
    <iconset resource="materialeditor.qrc">
     <normaloff>:/icons/icons/Compile.png</normaloff>:/icons/icons/Compile.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="materialeditor.qrc">
     <normaloff>:/icons/icons/New.png</normaloff>:/icons/icons/New.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="materialeditor.qrc"/>
 </resources>
 <connections/>
</ui>
